docs/resources/samples/automate-tasks-on-all-excel-files-in-folder.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Run a script on all Excel files in a folder'
```

Comment:
I think this only works on one workbook. We should include the flow (that I assume is in the video) before publishing this. Otherwise, it's misleading. Happy to tackle that in a separate PR, if you can create the issue.

Reply:
Tracking with feature 4924924